oauth/services/rhsso.py

```diff
@@ -34,7 +34,10 @@ def exchange_code_for_login(self, app_config, http_client, code, redirect_suffix
 
                 # 200 => Endpoint was hit successfully and user was found
                 # 400 => Endpoint was hit successfully but no user was found
-                if result.status_code == 200 and result.json().get("result", "") == "ERROR_EXPORT_CONTROL":
+                if (
```

Comment:
can you rebase for this

Reply:
it looks like i'm already on top of the latest commit - b21400b90d46ff1d6a989c2b3008769420bbd608